.github/workflows/deploy-package.yml

```diff
@@ -21,31 +21,31 @@ jobs:
       LIQUIBASE_VERSION: ${{ inputs.version }}
 
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
-
-    - name: Install Chocolatey
-      run: |
-        Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-    
-    - name: Install Ant
-      run: choco install ant
-      
-    - name: Build nupkg Chocolatey package
-      run: ant make-nuspec
-
-    - name: Choco pack
-      run: |
-        choco pack
-    
-    - name: Install OpenJDK8
-      run: choco install openjdk8
-
-    - name: Choco install
-      run: |
-        choco install liquibase --version="${{ inputs.version }}" --source="." --debug --verbose
-        liquibase --version
-
-    - name: Choco push
-      run: |
-        choco push liquibase.${{ inputs.version }}.nupkg --source "'https://push.chocolatey.org/'" --key "${{secrets.CHOCO_APIKEY}}" --debug --verbose
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Chocolatey
+        run: |
+          Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+
+      - name: Install Ant
+        run: choco install ant
+
+      - name: Build nupkg Chocolatey package
+        run: ant make-nuspec
+
+      - name: Choco pack
+        run: |
+          choco pack
+
+      - name: Install Temurin 17
+        run: choco install temurin17
```

Comment:
@jandroav @filipelautert are we ok with the latest version of temurin 17?  Looks like it is `17.0.10`

Reply:
I would go with `temurin17` to always have the latest fixes.

Reply:
I know we had an issue before with a particular minor version:
https://github.com/liquibase/liquibase/commit/bb5d00e8a24f795b5cdd85fe1ef322cd8c3d2e54

@filipelautert is this still an issue in the latest temurin?